doc/nrf/libraries/others/event_manager_profiler.rst

```diff
@@ -0,0 +1,109 @@
+.. _event_manager_profiler:
+
+Event Manager Profiler
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager Profiler is a piece of software that supports profiling for Event Manager.
+It serves as a linking layer between :ref`event_manager` and :ref:`profiler`.
+Together with Profiler, Event Manager Profiler enables logging and visualizing data for performance measurements, while the system is running.
+
+
+Configuration
+*************
+
+To use Event Manager Profiler, you must enable the following Kconfig option:
+
+* :kconfig:`CONFIG_EVENT_MANAGER_PROFILER` - This option enables the Event Manager Profiler.
+
+
+Implementation
+**************
+
+Event Manager Events are structured data types that are defined by the application and can contain additional data.
+The Event Manager Profiler expands each events data structure by adding to it profiler_info structure which holds information required by Profiler.
+Using those extra information Event Manager Profiler tracks usage of events.
+If EVENT_MANAGER_PROFILER_TRACE_EVENT_EXECUTION option is set Event Manager Profiler will track two extra events which mark start and stop of each events execution.
+
+
+Implementing profiling for Event Manager Events
+***********************************************
+
+If an application module is supposed to react to an event, your application must implement an event type, submit the event, and register the module as listener.
+Read the following sections for details.
+
+
+Profiling an event
+==================
+
+The Event Manager events can be profiled using :ref:`profiler`.
+Profiler allows you to observe the propagation of an event in the system, view the data connected with the event, or create statistics.
+
+To profile an event, you must complete the following steps:
+
+1. Enable the profiler using the :kconfig:`CONFIG_EVENT_MANAGER_PROFILER` Kconfig option.
```

Comment:
Enable profiling Event Manager events

Reply:
Yes that's true that it would be more precise and we can change it. :)  However I thought that in scope of event_manager_profiler every event is by default Event Manager event.

Reply:
The profiler uses events too. That may cause confusion at some point and it's why I think it's important to be precise here.